django_project/ford3/static/js/qualification_form.js

```diff
@@ -100,6 +100,71 @@ let customSelectChangedHandler = ($target, $hiddenListInput, selectedValue) => {
     $hiddenListInput.val(subjectListInputValue.join(','));
 };
 
+
+$('#add-qualification-event').click(function () {
+        addQualificationEvent();
+    }
+);
+
+
+ function addQualificationEvent(){
```

Comment:
It does look like the `addCampusEvent` function?
Is there a way to refactor?

Reply:
I would rather have them separate... these two just happen to look similar but they are for different forms. If either of the forms change I would just have to split them up again. I think I will refactor this on 3rd strike?